packages/fxa-content-server/app/scripts/views/security_events.js

```diff
@@ -0,0 +1,73 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import 'modal';
+import BaseView from './base';
+import SecurityEvent from '../../scripts/models/security-events';
+import Template from 'templates/security_events.mustache';
+
+let account;
+
+const View = BaseView.extend({
+  template: Template,
+  className: 'security-events',
+  viewName: 'security_events',
+
+  mustVerify: true,
+
+  events: {
+    'click #delete-events': '_deleteSecurityEvents',
+  },
+
+  beforeRender() {
+    account = this.getSignedInAccount();
+    if (!account) {
+      this.navigate('/signin');
+    }
+
+    return this._fetchSecurityEvents();
+  },
+
+  initialize(options = {}) {
+    if (options.securityEvents) {
```

Comment:
I'm not really sure if this function is needed? The before render will fetch the events.